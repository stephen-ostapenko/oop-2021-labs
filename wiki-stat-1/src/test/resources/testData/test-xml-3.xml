<mediawiki xmlns="http://www.mediawiki.org/xml/export-0.10/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mediawiki.org/xml/export-0.10/ http://www.mediawiki.org/xml/export-0.10.xsd" version="0.10" xml:lang="ru">
    <siteinfo>
        <sitename>Википедия</sitename>
        <dbname>ruwiki</dbname>
        <base>https://ru.wikipedia.org/wiki/%D0%97%D0%B0%D0%B3%D0%BB%D0%B0%D0%B2%D0%BD%D0%B0%D1%8F_%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0</base>
        <generator>MediaWiki 1.36.0-wmf.14</generator>
        <case>first-letter</case>
        <namespaces>
            <namespace key="-2" case="first-letter">Медиа</namespace>
            <namespace key="-1" case="first-letter">Служебная</namespace>
            <namespace key="0" case="first-letter" />
            <namespace key="1" case="first-letter">Обсуждение</namespace>
            <namespace key="2" case="first-letter">Участник</namespace>
            <namespace key="3" case="first-letter">Обсуждение участника</namespace>
            <namespace key="4" case="first-letter">Википедия</namespace>
            <namespace key="5" case="first-letter">Обсуждение Википедии</namespace>
            <namespace key="6" case="first-letter">Файл</namespace>
            <namespace key="7" case="first-letter">Обсуждение файла</namespace>
            <namespace key="8" case="first-letter">MediaWiki</namespace>
            <namespace key="9" case="first-letter">Обсуждение MediaWiki</namespace>
            <namespace key="10" case="first-letter">Шаблон</namespace>
            <namespace key="11" case="first-letter">Обсуждение шаблона</namespace>
            <namespace key="12" case="first-letter">Справка</namespace>
            <namespace key="13" case="first-letter">Обсуждение справки</namespace>
            <namespace key="14" case="first-letter">Категория</namespace>
            <namespace key="15" case="first-letter">Обсуждение категории</namespace>
            <namespace key="100" case="first-letter">Портал</namespace>
            <namespace key="101" case="first-letter">Обсуждение портала</namespace>
            <namespace key="102" case="first-letter">Инкубатор</namespace>
            <namespace key="103" case="first-letter">Обсуждение Инкубатора</namespace>
            <namespace key="104" case="first-letter">Проект</namespace>
            <namespace key="105" case="first-letter">Обсуждение проекта</namespace>
            <namespace key="106" case="first-letter">Арбитраж</namespace>
            <namespace key="107" case="first-letter">Обсуждение арбитража</namespace>
            <namespace key="446" case="first-letter">Education Program</namespace>
            <namespace key="447" case="first-letter">Education Program talk</namespace>
            <namespace key="828" case="first-letter">Модуль</namespace>
            <namespace key="829" case="first-letter">Обсуждение модуля</namespace>
            <namespace key="2300" case="first-letter">Гаджет</namespace>
            <namespace key="2301" case="first-letter">Обсуждение гаджета</namespace>
            <namespace key="2302" case="case-sensitive">Определение гаджета</namespace>
            <namespace key="2303" case="case-sensitive">Обсуждение определения гаджета</namespace>
            <namespace key="2600" case="first-letter">Тема</namespace>
        </namespaces>
    </siteinfo>
    <page>
        <title>Как прошел Сережин день</title>
        <ns>0</ns>
        <id>4</id>
        <redirect title="Заглавная страница" />
        <restrictions>sysop</restrictions>
        <revision>
            <id>237414</id>
            <parentid>22491</parentid>
            <timestamp>2004-08-09T04:36:57Z</timestamp>
            <contributor>
                <username>Maximaximax</username>
                <id>450</id>
            </contributor>
            <comment>redir</comment>
            <model>wikitext</model>
            <format>text/x-wiki</format>
            <text bytes="49" xml:space="preserve">Сережин день прошел неплохо. Не так хорошо, как мог бы, но и нельзя сказать, чтобы прямо уж плохо. Поэтому неплохо. Сережа написал целых две функции в командном проекте по ООП. Но это не главное. Главное заключается в том, что впервые за долгих полгода дистанционного обучения Сережа наконец-то пришел в любимый университет. К сожалению, это не означает того, что Сережу выпустили с дистанционки, но все-таки, это большой шаг на пути к старой нормальной жизни. Пожелаем Сереже удачи в выходе с дистанционки. P. S. А я тут недавно понял, что я сам настолько привык к текущей жизни, что вряд ли смогу нормально учиться очно. Это печально :(.</text>
            <sha1>fv75zcmgjvd2h4rn3jqc0yrlf07afkh</sha1>
        </revision>
    </page>
</mediawiki>