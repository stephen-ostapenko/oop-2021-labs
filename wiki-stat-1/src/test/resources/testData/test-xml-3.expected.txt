Как прошел Сережин день
Сережин день прошел неплохо. Не так хорошо, как мог бы, но и нельзя сказать, чтобы прямо уж плохо. Поэтому неплохо. Сережа написал целых две функции в командном проекте по ООП. Но это не главное. Главное заключается в том, что впервые за долгих полгода дистанционного обучения Сережа наконец-то пришел в любимый университет. К сожалению, это не означает того, что Сережу выпустили с дистанционки, но все-таки, это большой шаг на пути к старой нормальной жизни. Пожелаем Сереже удачи в выходе с дистанционки. P. S. А я тут недавно понял, что я сам настолько привык к текущей жизни, что вряд ли смогу нормально учиться очно. Это печально :(.
2004-08-09T04:36:57Z
49